components/subnavbar/index.tsx

```diff
@@ -0,0 +1,17 @@
+import Link from "next/link";
+import styles from "./subnavbar.module.scss";
+
+const SubNavbar = () => {
+  return (
+    <ul className={styles.container}>
+      <li>
+        <Link href="/series/current">Current</Link>
+      </li>
+      <li>
+        <Link href="/series/future">Future</Link>
+      </li>
+    </ul>
+  );
+};
+
```

Comment:
Similar to the above navbar component here <link> is not required, only <li> should be there. This subnav component can be used in the current & future route files under pages. 

return (
        <Navbar>
         <SubNavbar>
         <Lorel Ipsum content>
  )